doc/debugging.md

```diff
@@ -66,3 +66,35 @@ For WPF apps use `wpf`, e.g.(WPF debugging available only for react-native-windo
     "platform": "wpf"
 }
 ```
+
+## Debugging React Native apps with Hermes enabled
+
+Hermes is the new JavaScript engine optimized for running React Native apps on Android. It improves app performance and decreases app size.
+
+To enable Hermes for your app please follow [this official instructions](https://facebook.github.io/react-native/docs/hermes/).
```

Comment:
Please change the line to - 
``Click [here](https://facebook.github.io/react-native/docs/hermes/) to learn more about Hermes.``